File: php/utility/ResultBody.php
<?php
declare(strict_types=1);

// Ipify SDK utility: result_body

class IpifyResultBody
{
    public static function call(IpifyContext $ctx): ?IpifyResult
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}

File: .sdk/tm/rb/utility/result_body.rb
# Ipify SDK utility: result_body
module IpifyUtilities
  ResultBody = ->(ctx) {
    response = ctx.response
    result = ctx.result
    if result && response && response.json_func && response.body
      result.body = response.json_func.call
    end
    result
  }
end
